ogusa/output_plots.py

```diff
@@ -335,7 +335,6 @@ def ability_bar(base_tpi, base_params, reform_tpi,
     plt.ylabel(r'Percentage Change in ' + VAR_LABELS[var])
     if plot_title:
         plt.title(plot_title, fontsize=15)
-    plt.legend(loc=9, bbox_to_anchor=(0.5, -0.15), ncol=2)
```

Comment:
@jdebacker Why are you removing this legend in this figure? I don't see it anywhere else.

Reply:
There are no legend labels created for this bar plot and they'd be unnecessary anyway.  The line removed was creating some warnings about using a legend without any labels.

Reply:
Got it. Thanks.